WordPress/Classes/ViewRelated/Gutenberg/GutenbergFeaturedImageHelper.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import Gutenberg
+
+class GutenbergFeaturedImageHelper: NSObject {
+    fileprivate let post: AbstractPost
+    fileprivate let gutenberg: Gutenberg
+
+    let mediaIdNoFeaturedImageSet = 0
```

Comment:
As noted in other comments below, this could be a static let instead, allowing you to define it once here and easily use it in other files.